assets/multiqc_config.yml

```diff
@@ -1,7 +1,7 @@
 report_comment: >
-  This report has been generated by the <a href="https://github.com/nf-core/rnaseq" target="_blank">nf-core/rnaseq</a>
+  This report has been generated by the <a href="https://github.com/nf-core/rnaseq/releases/tag/dev" target="_blank">nf-core/rnaseq</a>
```

Comment:
yeah this is wrong in the template too, we have no releases for dev

Reply:
Yep, agree. We need to find a way to fix this or just remove it. Will keep it in for now as it will break linting.